modules/EG_Spectator_Mode/settings.sqf

```diff
@@ -19,4 +19,4 @@
 	Show_Entities_And_Locations_Lists = true;
 
 // Define where the spectator camera starts. (add a marker with the name inside the "")
-	Spectator_Marker = "SpectatorMarker";
\ No newline at end of file
+	Spectator_Marker = "";
```

Comment:
We should add that it starts on player death by default, but that a marker can be placed as the start place.